recipes/medcoupling/meta.yaml

```
@@ -0,0 +1,69 @@
+{% set name = "medcoupling" %}
+{% set version = "9.10.0" %}
+
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  - fn: 'medcoupling_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/medcoupling.git;a=snapshot;h=V{{ version.replace('.', '_') }};sf=tgz
+    sha256: 02087a4e59ffcdab6fc68460ffd86b03a80b5c0faa695b0bfb2142f1d0d3e5ea
+    patches:
+      - swig_big_file_compile_flag.patch
+  - fn: 'config_{{ version }}.tar.gz'
+    url: https://git.salome-platform.org/gitweb/?p=tools/configuration.git;a=snapshot;h=V{{ version.replace('.', '_') }};sf=tgz
+    folder: deps/config
+    sha256: d58f6749514ce49b00590a96d6dc9090a76a07c13690e2453403643afb40346c
+
+
+build:
+  skip: True  # [osx]
+  detect_binary_files_with_prefix: true
+  # string: aster_py{{ CONDA_PY }}_mpi_{{ mpi }}_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}   # [mpi != "nompi"]
+  # string: aster_py{{ CONDA_PY }}_nompi_h{{ PKG_HASH }}_{{ PKG_BUILDNUM }}           # [mpi == "nompi"]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - make                                      # [not win]
+    - ninja                                     # [win]
+    - cmake
+    - swig >=4.0.2
+
+  host:
+    - python
+    - numpy
+    - boost
+    - zlib
+    - libxml2
+    - libmed
+    - petsc                                     # [mpi != 'nompi']
+    - {{ mpi }}                                 # [mpi != 'nompi']
+
+  run:
+    - python
+    - numpy
```

```suggestion
    - {{ pin_compatible('numpy') }}
```

I think numpy finally got run_exports, but I think this still works.